src/context/WalletProvider/NativeWallet/NativePasswordRequired/NativePasswordRequired.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  Button,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  IconButton,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay
+} from '@chakra-ui/react'
+import { RawText } from 'components/Text'
+import { useForm } from 'react-hook-form'
+import { FaEye, FaEyeSlash } from 'react-icons/fa'
+
+import { useInitializeWalletFromStorage } from './hooks/useInitializeWalletFromStorage/useInitializeWalletFromStorage'
+import { useNativePasswordRequired } from './hooks/useNativePasswordRequired/useNativePasswordRequired'
+import { useStateToggle } from './hooks/useStateToggle/useStateToggle'
+
+export const NativePasswordRequired = () => {
+  const [showPw, toggleShowPw] = useStateToggle()
+  const {
+    handleSubmit,
+    register,
+    setError,
+    clearErrors,
+    formState: { errors, isSubmitting }
+  } = useForm()
+  const { onSubmit, isOpen, onClose } = useNativePasswordRequired({ setError, clearErrors })
+  useInitializeWalletFromStorage()
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} isCentered>
+      <ModalOverlay />
+      <ModalContent>
+        <Flex justifyContent='space-between' alignItems='center' position='relative'>
+          <ModalCloseButton ml='auto' borderRadius='full' position='static' />
+        </Flex>
+        <ModalHeader>Enter your password</ModalHeader>
+        <ModalBody>
+          <RawText mb={6} color='gray.500'>
+            Enter a password to encrypt your wallet. In order to securely store your keys we will
```

Comment:
translations plz

this should also say "enter a password to decrypt the wallet saved in your browser" or words to that effect